### PR TITLE
feat(atomic): add label prop on NoItems functional component

### PR DESCRIPTION
**Pull request by fbeaudoincoveo**
https://coveord.atlassian.net/browse/KIT-3524

### CODE REVIEW

File: packages/atomic/src/locales.json [hunk 1]
```diff
@@ -8289,5 +8289,59 @@
...
+    "ko": "제품 없음",
+    "nl": "Geen producten",
+    "no": "Ingen produkter",
+    "pl": "Brak produktów",
+    "pt": "Sem produtos",
+    "pt-BR": "Sem produtos",
+    "ru": "Нет продуктов",
+    "sv": "Inga produkter",
+    "th": "ไม่มีสินค้า",
+    "tr": "Ürün yok",
```
Comment (dmgauthier) [inline packages/atomic/src/locales.json:8315]: Sounds like Tolkien Mordor language.  I dig it.

**Review by dmgauthier — APPROVED** [review]

File: packages/atomic/src/locales.json (hunk 1 above)
Comment (louis-bompart) [reply packages/atomic/src/locales.json:8315]: We're missing `tlh` support :/

**Review by louis-bompart — APPROVED** [review]

File: packages/atomic/src/components/commerce/atomic-commerce-no-products/e2e/page-object.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ export class NoProductsPageObject extends BasePageObject<'atomic-commerce-no-pro
   message(query?: string) {
     return this.page.locator('[part="no-results"]', {
       hasText: query
-        ? `We couldn't find any product for “${query}”`
+        ? `We couldn't find any product for ${query}}"`
```
Comment (dmgauthier) [inline packages/atomic/src/components/commerce/atomic-commerce-no-products/e2e/page-object.ts]: You sure about this ? This single double quote is suspicious

**Review by dmgauthier — APPROVED** [review]

File: packages/atomic/src/components/commerce/atomic-commerce-no-products/e2e/page-object.ts (hunk 2 above)
Comment (fbeaudoincoveo) [reply packages/atomic/src/components/commerce/atomic-commerce-no-products/e2e/page-object.ts]: Fixed 😄

**Review by dmgauthier — APPROVED** [review]

**Pull request merged**